src/gc/impl/conservative/gc.d

```diff
@@ -1214,7 +1214,7 @@ class ConservativeGC : GC
         size_t bsize = 0;
 
         //debug(PRINTF) printf("getStats()\n");
-        memset(&stats, 0, GCStats.sizeof);
+        memset(&stats, 0, core.memory.GC.Stats.sizeof);
```

Comment:
Nitpick: remove the `memset` because it is redundant for `out` parameters. (See https://dlang.org/spec/function.html#parameters)